cdc/api/v2/model.go

```diff
@@ -1024,6 +1046,16 @@ type ChangeFeedInfo struct {
 	TaskStatus     []model.CaptureTaskStatus `json:"task_status,omitempty"`
 }
 
+// SyncedStatus describes the detail of a changefeed's synced status
+type SyncedStatus struct {
+	Synced           bool   `json:"synced"`
+	SinkCheckpointTs string `json:"sink_checkpoint_ts"`
```

Comment:
Why use string instead of time.Time? Ref: https://github.com/pingcap/tiflow/blob/master/cdc/api/v2/changefeed.go#L299C10-L302

Reply:
Good suggestion! I change to use time.Time now.